### PR TITLE
Fixed quotes and missing lines

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: src/translations/qterminal_el.desktop.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
+Desktop Entry/Name: "QTerminal"
+Desktop Entry/GenericName: "Προσομοιωτής τερματικού""
```
Comment (yan12125) [inline src/translations/qterminal_el.desktop.yaml:2]: There are duplicated quotes - I pushed a fix directly to git master.

File: src/translations/qterminal_pt.desktop.yaml [hunk 2]
```diff
@@ -1,2 +1,4 @@
+Desktop Entry/Name: "Qterminal"
+Desktop Entry/GenericName: "Emulador de terminal"
```
Comment (yan12125) [inline src/translations/qterminal_pt.desktop.yaml:2]: Hmm, authorship by `Sérgio Marques <smarquespt@gmail.com>` is lost and weblate is confused (https://github.com/lxqt/qterminal/commit/dce787632f5558ceb08fe82245dd1b90c8722123) after adding new translation here.

Comment (stefonarch) [reply src/translations/qterminal_pt.desktop.yaml:2]: I found several commits already merged with the missing quotes.
The doubled ones come from my setting in Featherpad, should change it, some slipped through I see...

Comment (yan12125) [reply src/translations/qterminal_pt.desktop.yaml:2]: I'm afraid not if we don't want force pushes to this git repository, so I Ieave a comment here.

**Pull request merged**